src/Sylius/Bundle/AdminBundle/Resources/config/grids/promotion.yml

```diff
@@ -10,15 +10,33 @@ sylius_grid:
                     type: string
                     label: sylius.ui.code
                 name:
-                    type: string
+                    type: twig
                     label: sylius.ui.name
-                usage_limit:
-                    type: string
-                    label: sylius.ui.usage_limit
-                used:
-                    type: string
-                    label: sylius.ui.used
+                    path: .
+                    options:
+                        template: SyliusAdminBundle:Grid/Field:nameAndDescription.html.twig
+                couponBased:
+                    type: twig
+                    label: sylius.ui.coupons
+                    options:
+                        template: SyliusAdminBundle:Grid/Field:yesNo.html.twig
```

Comment:
@pjedrzejewski What are your opinion about to use the name of the template like the field type. For example, yesNo.html.twig => boolean.html.twig, etc...